gradle/versions.gradle

```diff
@@ -41,9 +41,9 @@ dependencyManagement {
 
     dependency 'javax.activation:activation:1.1.1'
 
-    dependency 'org.apache.logging.log4j:log4j-api:2.17.0'
-    dependency 'org.apache.logging.log4j:log4j-core:2.17.0'
-    dependency 'org.apache.logging.log4j:log4j-slf4j-impl:2.17.0'
+    dependency 'org.apache.logging.log4j:log4j-api:2.17.1'
```

Comment:
be better to use a dependency group for the version

Reply:
done.